tests/yang/YANGBearTest.py

```diff
@@ -0,0 +1,45 @@
+from bears.yang.YANGBear import YANGBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+COALA_MODEL = """
+module coala-model {
+
+  namespace "http://coala.io/yang/coala-model";
+  prefix coala;
```

Comment:
These string literals looks ugly....breaks line of flow
Use this https://docs.python.org/2/library/textwrap.html#textwrap.dedent
This is not a compulsory change..It's just a suggestion :wink: 
Rest LGTM :)

Reply:
@damngamerz It is compulsory! Believe me ;)

Reply:
@damngamerz Actually it's compulsory to move them to separate `.yang` files! :)

Reply:
Much better :) :+1: